assignments/prototypes.js

```diff
@@ -22,6 +32,16 @@
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function CharacterStats(stats){
+  GameObject.call(this, stats);
+  this.healthPoints = stats.healthPoints;
+  this.name= stats.name;
+}
+
```

Comment:
Don't forget, we have to add this to give the CharacterStats constructor the properties from GameObject.prototype.

```suggestion
  CharacterStats.prototype = Object.create(GameObject.prototype);

```